OpenDreamServer/Dream/Objects/DreamList.cs

```diff
@@ -67,26 +67,25 @@ class DreamList : DreamObject {
         }
 
         public virtual DreamValue GetValue(DreamValue key) {
-            if (key == DreamValue.Null) return DreamValue.Null; //TODO: Null key
+            if (key == DreamValue.Null) // x = List[null]
+                key = new(String.Empty);
```

Comment:
This makes `list[null]` and `list[""]` the same thing, which is incorrect.

Reply:
What do you think then? We'd have to use some cursed placeholder value?